populate.json

```diff
@@ -382,7 +382,8 @@
         "model": "nexong.homeDocument",
         "pk": 1,
         "fields": {
-            "title": "documentodecasa",
+            "title": "documentoDeCasa",
```

Comment:
Could you change this title to something more descriptive? Like `balance_cuentas_2023`.